docs/design-docs/kafka-monitoring-tools/kafka-monitoring-tools.md

```diff
@@ -0,0 +1,126 @@
+- It does not monitor anything related to the health of the brokers
+- Possibility to set up alerts
+
+## 11. KAFKA DROP 3 [link](https://github.com/obsidiandynamics/kafdrop)
+
+- Opensource project, Apache-2.0 License, reboot of Kafdrop 2.x
+- Monitoring tool (including view consumer offset)
+- Out of the box UI
+- No possibility to set up alerts
+
+## 12. KAFKA MONITOR [link](https://github.com/linkedin/kafka-monitor)
+
+- Opensource project, Apache-2.0 License
+- Kafka monitor is a framework to implement and execute long-running Kafka system tests in a real cluster
+- It plays a role as a passive observer and reports what it observes (broker availability, produce/consume latency, etc) by emitting metrics. In other words, it pretends to be a Kafka user and keeps reporting metrics from the user's PoV
+- It is more a load generation and reporting tool
+- UI does not exist
+- No possibility to set up alerts
+
+## 13. OTHERS
+
+Things like on the list below are there as well, but usually such smaller projects and have little or no development activity:
+
+- [doctorK](https://github.com/pinterest/DoctorK)
+- [kafdrop](https://github.com/obsidiandynamics/kafdrop)
+- [kafka-offset-monitor](https://github.com/Morningstar/kafka-offset-monitor)
+
+## 14. CONCLUSIONS
+
+Currently in Epiphany monitoring and getting metrics from Kafka are based one:
+- Kafka Exporter [link](https://github.com/danielqsj/kafka_exporter)
+- JMX Exporter [link](https://github.com/prometheus/jmx_exporter)
+- Prometheus [link](https://prometheus.io/)
+- Grafana [link](https://grafana.com/)
+
+In real scenarios, based on some use cases and opinions from internal teams:
+- Kafka Exporter is used in order to get consumer offset and lag
+- JMX Exporter is used in order to get some standard broker's metrics such as cpu, memory utilization and so on
+- Problems with the administration of the leader election are reported
+
+If it is possible to pay for a commercial license, Confluent, Lenses and Sematext offer more rich functionality compared to the other monitoring tools and they are very similar.
+
+As far as the open source project is considered:
+- LinkedIn Cruise Control looks like the winner. Provides not only managing and monitoring Kafka cluster but also some extra features such as rebalancing, anomaly detection or self-healing
+- Yahoo Cluster Manager looks like good competitor but only for managing and monitoring Kafka cluster. However in compare to Cruise Control, during the installation I met some issues and I was not able to recieve some consumer data and a few issues are already reported in official repository related to my problem [link](https://github.com/yahoo/CMAK/issues/641). I would not recommend this tools since it looks like the project does not have good spirit of open source software
```

Comment:
```suggestion
- Yahoo Cluster Manager looks like good competitor but only for managing and monitoring Kafka cluster. However in compare to Cruise Control, during the installation I met some issues and I was not able to recieve some consumer data and a few issues are already reported in official repository related to my problem [link](https://github.com/yahoo/CMAK/issues/641). I would not recommend these tools since it looks like the project does not have good spirit of open source software
```